### ISSUE TITLE
Model needs a program

### ISSUE THREAD

**Issue by gingerizer** [uber/nebula.gl#114]
Attempting to use nebula for the first time.  Trying to integrate it into an existing project that uses DeckGL and StaticMap from react-map-gl.

When I use the code from the example [Hero.js](https://github.com/uber/nebula.gl/blob/8f787f68060f54e2a26ca002cc4a0ce2656993d6/website/src/components/Hero.js)  I'm unable to get it to work.

The errors I'm getting are:
```
deck: error while initializing invalid
  Error: Model needs a program
  at assert (assert.js?9b39:12)
  at Model._createProgram (model.js?5534:554)
  at Model.initialize (model.js?5534:86)
  at SolidPolygonLayer._getModels (solid-polygon-layer.js?3294:332)
  at SolidPolygonLayer.updateState (solid-polygon-layer.js?3294:224)
  at SolidPolygonLayer._updateState (layer.js?49b1:715)
  at SolidPolygonLayer._initialize (layer.js?49b1:684)
  at LayerManager._initializeLayer (layer-manager.js?1ae6:653)
  at LayerManager._updateSublayersRecursively (layer-manager.js?1ae6:582)
```
```
Uncaught TypeError: Cannot read property 'render' of undefined
  at PathLayer.draw (path-layer.js?dabf:219)
  at eval (layer.js?49b1:805)
  at withParameters (context-state.js?94fb:238)
  at PathLayer.drawLayer (layer.js?49b1:804)
  at drawLayerInViewport (draw-layers.js?e0dc:305)
  at eval (draw-layers.js?e0dc:254)
  at Array.forEach (<anonymous>)
  at drawLayersInViewport (draw-layers.js? e0dc:226)
  at eval (draw-layers.js?e0dc:112)
  at Array.forEach (<anonymous>)
```

Not sure if I'm missing something but can't figure out what is incorrect and assume that the example should work as is.

**Comment by supersonicclay** [comment]
Can you paste an example of your code here?

**Comment by georgios-uber** [comment]
What version of deck.gl are you using?

Can you provide your whole yarn.lock so I can see all versions of libraries?

**Comment by gingerizer** [comment]
I started going through the process of setting something up on codesandbox and it highlighted the issue of multiple versions of deck.gl in the project configuration (deck.gl@6.2.0 and deck.gl@6.0.5).    

I was trying to use the same versions in my project (nebula.gl@0.6.3) and was not able to due to the conflict.   In order to resolve the version conflict I had to chagne the version of nebula.gl to 0.6.2.   

Went back to my project and made the same change and I am not seeing the layers rendering correctly on the map.
